Update wallet-core-usage.md (#96)
trustwallet/developer: wallet-core/wallet-core-usage.md

Here is the Swift sample code for signing a real world Bitcoin Cash [transaction](https://blockchair.com/bitcoin-cash/transaction/96ee20002b34e468f9d3c5ee54f6a8ddaa61c118889c4f35395c2cd93ba5bbb4)

```swift

> let utxoTxId = Data(hexString: "050d00e2e18ef13969606f1ceee290d3f49bd940684ce39898159352952b8ce2")! // latest tx id for sender, get from blockchain "txix"

Reviewer: not always latest tx, `get from blockchain` blockchain.com api?

let privateKey = PrivateKey(data: Data(hexString: "7fdafb9db5bc501f2096e7d13d331dc7a75d9594af3d251313ba8b6200f4e384")!)!
let address = CoinType.bitcoinCash.deriveAddress(privateKey: privateKey)

let utxo = BitcoinUnspentTransaction.with {
    $0.outPoint.hash = Data(utxoTxId.reversed()) // reverse of UTXO tx id, Bitcoin internal expects network byte order

>     $0.outPoint.index = 2                        // outpoint index of this this UTXO // index from latest transaction vout array, need get index for transaction which was connected to send back balance to sender walled

Reviewer: ```suggestion
    $0.outPoint.index = 2                        // outpoint index of this this UTXO, get from vout property
```

    $0.outPoint.sequence = UINT32_MAX

>     $0.amount = 5151                             // value of this UTXO - need get from value property vout[index]

Reviewer: ```suggestion
    $0.amount = 5151                             // value of this UTXO - get from vout[index] value property
```
